instaclone/src/components/PostContainer/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import './CommentSection.css';
+import ActualComment from './ActualComment';
+
+class CommentSection extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			comments: this.props.comment,
+			newComment: '',
+			likes: 0
+		};
+	}
+	commentInput = e => {
+		this.setState({ newComment: e.target.value });
+	};
+
+	addNewComment = event => {
+		event.preventDefault();
+		const userName = 'Michael';
+		const nComment = { text: this.state.newComment, username: userName };
+		const commentList = this.state.comments.slice();
+		commentList.push(nComment);
```

Comment:
Good job, breaking out your variables here. It very clear what is happening as a new comment get created and eventually added to the commentList. 👍